### PR TITLE
Add guide: How to optimize Netdata's performance

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

A new guide to help users optimize performance of the Netdata Agent. This PR takes most of its information from a few older, disparate files, condenses them, and deletes/deprecates them.

##### Component Name

area/docs

### CODE REVIEW

**Comment by joelhans** [comment]
[How to maximize Netdata's performance](https://trello.com/c/dUuZxj2h/70-how-to-maximize-netdatas-performance)

**Review by aabatangle — APPROVED** [review]

File: docs/guides/configure/performance.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+## What affects Netdata's performance?
+
+Netdata's performance is primarily affected by **data collection/retention** and **clients accessing data**. 
+
+You can configure almost all aspects of data collection/retention, and certain aspects of clients accessing data. For
+example, you can't control how many users might be viewing a local Agent dashboard, [viewing an
+infrastructure](/docs/visualize/overview-infrastructure.md) in real-time with Netdata Cloud, or running [Metric
+Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations).
+
+The Netdata Agent runs with the lowest possible [/daemon/README.md#netdata-process-scheduling-policy), which is `nice
```
[the remark this answers is not in the document]
Comment (joelhans) [reply docs/guides/configure/performance.md]: Something got inadvertently deleted! Fixed this.

**Comment by joelhans** [comment]
@OdysLam Ping! Your review would be appreciated—this is scheduled to publish tomorrow.

**Review by aabatangle — APPROVED** [review]

File: docs/guides/configure/performance.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+All the settings are found in the `[global]` section of `netdata.conf`:
+
+```conf
+[global]
+	memory mode = dbengine
+	page cache size = 32
+  dbengine multihost disk space = 256
+```
+
+## Run Netdata behind Nginx
```
Comment (OdysLam) [inline docs/guides/configure/performance.md:156]: I thnk is weird that we say that in order to minimise the footprint of netdata you need it to run behind nginx. Do we have any numbers to back that claim?

I know that it's more performant for the reasons you mention, but I am not sure if for a very resource constrainted environment, it's better to run nginx or just say that I won't access the dashboard from many browsers at the same time.

**Review by OdysLam — APPROVED** [review]

**Pull request merged**